client/app/lib/components/redux-form/SingleFileInput.jsx

```diff
@@ -78,13 +86,21 @@ class SingleFileInput extends React.Component {
   }
 
   onDrop = (files) => {
+    const { isDropFileSupported } = this.props;
+    if (this.isFunction(isDropFileSupported) && !isDropFileSupported(files[0])) {
```

Comment:
Is there a need to check here if `isDropFileSupport` prop is a function?
It has already been declared as `PropTypes.func`.

Reply:
Ok, will remove it.